laguinho/routes/datasets.py

```diff
@@ -11,6 +11,14 @@ def publish():
     datasets_metadata.append(result)
     return jsonify(result), 201
 
-@datasets.route("/datasets", methods=['GET'])
+@datasets.route("/datasets/", methods=['GET'])
 def get_datasets():
     return jsonify(datasets_metadata)
+
+@datasets.route("/datasets/<name>", methods=['GET'])
+def get_datasets_by_name(name):
+  search_result = list(
+    filter(lambda dataset: dataset['name'] == name, datasets_metadata))
+  
+  result = search_result[0] if len(search_result) > 0 else None
```

Comment:
Se a issue anterior for resolvida com um dict, essa checagem pode ser removida, mas caso contrário, seria bom lançarmos um erro aqui

Reply:
Vale a pena mudar para dicionário agora, tendo em vista que nós vamos substituir por um BD futuramente?